01-js-intro-and-jquery/index.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function (){
+	$('#sec').hide();
+	$('#sec').fadeIn(2000);
+	$('.alias').focus();
```

Comment:
The focus should happen when the fadeIn finishes, therefore as a callback, in this case it will happen when fadeIn starts.